codejail/proxy.py

```diff
@@ -57,10 +59,12 @@ def run_subprocess_through_proxy(*args, **kwargs):
             log.exception("Proxy process failed")
             # Give the proxy process a chance to die completely if it is dying.
             time.sleep(.001)
+            last_exec_info = sys.exc_info()
             continue
 
     # If we finished all the tries, then raise the last exception we got.
-    raise
+    if last_exception:
```

Comment:
Where are we setting this to values other than `None`? Should `last_exec_info` above be `last_exception`?

Reply:
yep, missed that in the re-name.